toolkit/scss/forms/_summary.scss

```diff
@@ -33,71 +33,86 @@
   padding-left: 75%;
   position: relative;
   z-index: 10;
+  font-weight: bold;
```

Comment:
If the `copy-19` above is changed to `bold-19` we won't need this line. Would be worth doing I think.